astroquery/linelists/jplspec/core.py

```diff
@@ -19,18 +23,21 @@ def data_path(filename):
     return os.path.join(data_dir, filename)
 
 
-@async_to_sync
```

Comment:
Can we remove the async methods as they are not really server async? Or if there is server async available, follow the new API and use the `async_job` kwargs instead?